### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Onyinye91-ctrl** [OpenMined/courses#140] [Introduce Yourself]
Hello Everyone, Nice to be here. I am Sandra from Nigeria, a Machine Learning Engineer Enthusiast. Currently working as a Data Analyst. Got to know about this course from a slack group i belong to. I hope to learn one or two, Thanks for the opportunity

**Comment by yemikifouly** [comment]
Hi Sandra from Nigeria! 🙂 Yemi from Benin here! Welcome to OpenMined and to the discussion board! Have you started the course yet? Also, I am sure that you will learn a lot more than a thing or two. 😀

**Reply by Onyinye91-ctrl** [reply]
Wow!! Thank You for the Warm welcome. will be starting today.
